packages/terra-form-input/src/FormInput.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './FormInput.scss';
```

Comment:
With renaming the JSX file to Input.jsx, let's also rename the scss file to `Input.scss`

Reply:
Done 0fa82cf1ef4966c97404188b2b229c8f8385fb2b